charts/newrelic-infra-operator/templates/secret.yaml

```diff
@@ -1,12 +1,12 @@
-{{- $licenseKey := include "newrelic-infra-operator.licenseKey" . -}}
+{{- $licenseKey := include "newrelic.common.license._licenseKey" . -}}
```

Comment:
Removes this file completely and copy this from `nri-kubernetes`: https://github.com/newrelic/nri-kubernetes/blob/main/charts/newrelic-infrastructure/templates/secret.yaml

Is the common library who will template this secrets (if needed)